dune.opam

```diff
@@ -11,7 +11,6 @@ depends: [
   "base-threads"
 ]
 build: [
-  # opam 2 sets OPAM_SWITCH_PREFIX, so we don't need a hardcoded path
```

Comment:
Why was this comment removed?

Reply:
Hm was this removed due to the rewriting of the opam file removing comments. We should test that specifically to make sure comments are preserved